apps/help-center/hooks/use-location.js

```diff
@@ -0,0 +1,13 @@
+import { privateApis as routerPrivateApis } from '@wordpress/router';
+import { getUnlock } from '../utils';
+
+const unlock = getUnlock();
+
+let useLocation;
```

Comment:
Oops...we should provide the default one to avoid it unavailable

```
let useLocation: () => Location | null = () => null;
```

Reply:
Yeah... I just saw this. My bad! Fix incoming.

Reply:
Fix here: #96046